blockchain/types/anchoring_data.go

```diff
@@ -60,14 +60,14 @@ func (data *AnchoringDataLegacy) GetBlockNumber() *big.Int {
 }
 
 type AnchoringDataInternalType0 struct {
-	BlockHash     common.Hash
-	TxHash        common.Hash
-	ParentHash    common.Hash
-	ReceiptHash   common.Hash
-	StateRootHash common.Hash
-	BlockNumber   *big.Int
-	BlockCount    *big.Int
-	TxCount       *big.Int
+	BlockHash     common.Hash `json:"BlockHash"`
+	TxHash        common.Hash `json:"TxHash"`
+	ParentHash    common.Hash `json:"ParentHash"`
+	ReceiptHash   common.Hash `json:"ReceiptHash"`
+	StateRootHash common.Hash `json:"StateRootHash"`
+	BlockNumber   *big.Int    `json:"BlockNumber"`
+	BlockCount    *big.Int    `json:"BlockCount"`
+	TxCount       *big.Int    `json:"TxCount"`
```

Comment:
We need to establish a case naming convention for json. 
Currently, each package/file has different conventions. 
As far as I know, snake cases are commonly used for json.

Reply:
I think this data is come from below `header`.
So I will camel case this also. 
```
type Header struct {
	ParentHash  common.Hash    `json:"parentHash"       gencodec:"required"`
	Rewardbase  common.Address `json:"reward"           gencodec:"required"`
	Root        common.Hash    `json:"stateRoot"        gencodec:"required"`
```